react-redux-boilerplate/README.md

```diff
@@ -0,0 +1,30 @@
+# react-redux-boilerplate
+
+Want to create a React/Redux app? Start here! Follow this README and the associated code comments
+to gain a better understanding of the mindset that drives React/Redux frontend programming, and how
+all these tools tie together.
+
+For an overview of React and Redux, as well as other tools we use for frontend development at
+Clever, see our frontend overview. TODO: LINK HERE
+
+## Application structure
+
+This project has 4 subdirectories:
+
+* `store/`: **State management**. Contains the single Redux store, which stores and organizes all
+    application state. When an "action" occurs, Redux `reducers` get called. Reducers are pure
+    functions that take in the current state and the action, and output the resulting new state.
+    Thus, the store determines how state is stored within it, and how the state can change as
+    different actions occur.
+* `actions/`: **Defining state transitions**: Contains all the possible actions that can occur in an
+    application. Actions just plain Javascript objects that contain the `type` of the action, and
+    any metadata the `store` needs to respond to that action.
+* `components/`: **Visual display**. React `Component`s that are **presentational** in nature—that
+    is, they do not specify how to manage state, but merely describe what HTML should be outputted
+    given their input properties.
```

Comment:
I'm in favor of leaving component state out of this example and suggest using a Redux store.  But, I do think we should come to an agreement/guide as to what when to use one or the other.